src/basic/syscall-list.txt

```diff
@@ -95,6 +95,7 @@ fsopen
 fspick
 fstat
 fstat64
+fstatat
```

Comment:
Hmm? `fstatat` isn't a syscall on any architecture?

Reply:
I am not sure. I just ran `ninja update-syscall-tables`.

Reply:
Maybe also add it to the `file-system` seccomp set then?

Reply:
> Maybe also add it to the `file-system` seccomp set then?

Done.